src/assets/thirdPartyServices/snet/yolov3_object_detection/index.js

```diff
@@ -4,40 +4,50 @@ import Button from "@material-ui/core/Button";
 import Slider from "@material-ui/lab/Slider";
 
 import SNETImageUpload from "../../standardComponents/SNETImageUpload";
+import { Detector } from "./object_detection_pb_service";
+
+
+const initialUserInput = {
+  methodName: "Select a method",
+  img_path: undefined,
+  confidence: 0.5,
+  img_path: "",
+  methodName: "detect",
+};
 
 export default class YOLOv3ObjectDetection extends React.Component {
   constructor(props) {
     super(props);
     this.submitAction = this.submitAction.bind(this);
-    this.handleServiceName = this.handleServiceName.bind(this);
+
+    //this.handleServiceName = this.handleServiceName.bind(this);
     this.handleFormUpdate = this.handleFormUpdate.bind(this);
     this.getImageURL = this.getImageURL.bind(this);
-    this.getServiceMethods = this.getServiceMethods.bind(this);
+    //this.getServiceMethods = this.getServiceMethods.bind(this);
```

Comment:
Commented code can be deleted from the entire file.

Reply:
I just kept them for reference during the test. Once all the test is done we can delete the commented code.